pom.xml

```diff
@@ -0,0 +1,373 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<groupId>org.bahmni.module</groupId>
+	<artifactId>bahmni-events</artifactId>
+	<version>1.0.0-SNAPSHOT</version>
+	<packaging>pom</packaging>
+	<name>Bahmni Events</name>
+	<description>Bahmni events module to send business events which can be used by external services</description>
+	<url>https://demo.mybahmni.org/bahmni/home/index.html</url>
+
+	<licenses>
+		<license>
+			<name>GNU AFFERO GENERAL PUBLIC LICENSE</name>
+			<url>https://www.gnu.org/licenses/agpl-3.0.txt</url>
```

Comment:
Here you have mentioned AGPL3. Then maybe add the same LICENSE.md file also.

Reply:
https://choosealicense.com/licenses/agpl-3.0/

Reply:
I have copied from openmrs appointment module. Will that work ?

Reply:
Added the file